package.json

```diff
@@ -1,11 +1,12 @@
 {
   "name": "@oclif/plugin-search",
-  "version": "1.1.4",
+  "version": "1.1.5-dev.2",
```

Comment:
will need to revert before merge - just noting

Reply:
`create-github-release` automatically handles that for us